### PR TITLE
scripts: profiler: Add stats calculation

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds ProcessedEvents and StatsNordic modules. Modules are used to calculate time intervals statistics between events occurences.

### CODE REVIEW

File: scripts/profiler/processed_events.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          (self.event_processing_end_id is None):
+            self.tracking_execution = False
+            for ev in self.raw_data.events:
+                self.tracked_events.append(TrackedEvent(ev, None, None))
+            return
+
+        for i in range(0, len(self.raw_data.events)):
+            if self.raw_data.events[i].type_id == self.event_processing_start_id:
+                self.start_event = self.raw_data.events[i]
+                for j in range(i - 1, -1, -1):
```
Comment (Qbicz) [inline scripts/profiler/processed_events.py]: is it possible that you will get to the `j` loop when `i` is `0`? If yes, you will get `-1` index and process some element twice

File: scripts/profiler/stats_nordic.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+import csv
+
+class EventState(Enum):
+    SUBMIT = 1
+    PROC_START = 2
+    PROC_END = 3
+
+class StatsNordic():
+    def __init__(self, events_filename, events_types_filename, log_lvl):
+        self.data_name = events_filename[0:events_filename.find('.')]
```
Comment (Qbicz) [inline scripts/profiler/stats_nordic.py]: What about using `events_filename.split('.')[0]` ?

Comment (MarekPieta) [reply scripts/profiler/stats_nordic.py]: done

File: scripts/profiler/processed_events.py (hunk 1 above)
Comment (MarekPieta) [reply scripts/profiler/processed_events.py]: Ok, we may process events starting from i = 1

**Comment by MarekPieta** [comment]
I made requested changes

**Pull request merged**